### ISSUE TITLE
Is it possible to use newspapper3k on files?

### ISSUE THREAD

**Issue by IKetchup** [codelucas/newspaper#918]
I currently working on a project which constist in extracting informations for repport. I would like to use newspaper for this but as for now I have only seen newspaper used with online article. Is it possible to use newspapper3k on html file or txt files ?

Thanks in advance for any informations about this topic.

**Comment by johnbumgarner** [comment]
Yes.  Please reference the section [Extraction from offline HTML files](https://github.com/johnbumgarner/newspaper3_usage_overview#extraction-from-offline-html-files) from my Newspaper3K usage document.
